docker-compose.yml

```
@@ -0,0 +1,31 @@
+    volumes:
+      - grafana_data:/var/lib/grafana
+    environment:
+      - GF_SECURITY_ADMIN_PASSWORD=${GF_ADMIN_PASSWORD}
+      - GF_USERS_ALLOWS_SIGNUP=false
+    depends_on:
+      - prometheus
+    restart: unless-stopped
+
+volumes:
+  prometheus_data:
+  grafana_data:  
```

YAMLlint (1.35.1)

[warning] too many spaces after colon

(colons)